### PR TITLE
Translate Tanks of Freedom to Spanish

### PR DESCRIPTION
**Pull request by xexio**
Spanish translation of Tanks of Freedom

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Tanks of Freedom
+# Tanks of Freedom - Added Spanish translation
```
Comment (w84death) [inline README.md]: This is the title of the readme file. Do not change it's title like that.

File: README.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ For online funcionalities a ToF server is required. Sources can be found in a se
 ### gfx
    - w84death http://krzysztofjankowski.com licensed under MIT
    - Free font Courier Prime http://www.quoteunquoteapps.com/courierprime/ licensed under OFL
-   - NotoSans font licensed under Apache License 2.0
+   - NotoSans fonr licensed under Apache License 2.0
```
Comment (w84death) [inline README.md]: why? or mistake?

File: README.md (hunk 1 above)
Comment (xexio) [reply README.md]: I reverted the change

File: README.md (hunk 2 above)
Comment (xexio) [reply README.md]: I reverted the change

**Comment by czlowiekimadlo** [comment]
I have integrated all new translation PRs, now you should be able to pull new commits and run the game from editor with ES language available in options. Merging script complained, that some labels might be too long:
LABEL_AUTO_FOLLOW, 
LABEL_WORKSHOP_SELECTED_TOOL, 
LABEL_REQUIES_RESTART, 
LABEL_PREVIOUS_UNIT, 
LABEL_NEXT_UNIT, 
LABEL_SELECTOR_MOVEMENT, 
LABEL_REFRESHING_MATCHES, 
LABEL_MATCH_FORFEIT, 
LABEL_CLEAR_SLOT, 
LABEL_WORKSHOP_CLEAR_MAP, 
LABEL_TOOLTIPS, 
LABEL_NO_MORE_MOVES

Sometimes this is not an issue, other times it may break some layouts. Best way to verify is to see if they are significanly longer than their EN counterparts.

Otherwise, thank you for the awesome contribution!

**Pull request merged**